src/Tool/Tool.csproj.orig

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <PlatformTarget>x64</PlatformTarget>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
It looks like this is an old file kept for reference, so it doesn’t technically need to be updated. That said, it shouldn’t hurt.

Reply:
I don't know why it's kept around - I reverted the change.